packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue

```diff
@@ -49,7 +55,14 @@
             <component
                 :is="errorBoundary"
                 tier="3">
-                <slot name="new-label" />
+                <!-- TODO - we want to translate this within the component using i18n.
+                For now we'll just need to pass down a translated string from the consuming site -->
+                <restaurant-badge
+                    v-if="newBadgeText"
+                    :is-large="true"
+                    :text="newBadgeText"
+                    :background-colour="'#e5faef'"
+                    :text-colour="'#017A39'" />
```

Comment:
Consistency in casing please:
```suggestion
                    :background-colour="'#e5faef'"
                    :text-colour="'#017a39'" />
```